src/components/VaccinationsTable/VaccinationsTable.tsx

```diff
@@ -47,26 +52,28 @@ const Column: React.FC<{ listHeader: string; locations: any[] }> = ({
   );
 };
 
-const VaccinationsTable: React.FC<{}> = () => {
-  const highestRanking = getHighestRankingStates(5);
-  const lowestRanking = getLowestRankingStates(5);
+const VaccinationsTable: React.FC<{ mapView: MapView }> = ({ mapView }) => {
+  const highestRanking = getHighestRankingRegions(5, mapView);
+  const lowestRanking = getLowestRankingRegions(5, mapView);
+
+  const buttonLabel = <>See all {mapView.toLowerCase()}</>;
```

Comment:
Just FYI (not worth changing)- If `ExpandableContainer` accepted `ReactNode` instead of `ReactElement` for the labels then we could just use plain strings here instead of having to wrap in `<>...</>` since `ReactNode` is more encompassing and includes `string`:
```
    type ReactText = string | number;
    type ReactChild = ReactElement | ReactText;

    interface ReactNodeArray extends Array<ReactNode> {}
    type ReactFragment = {} | ReactNodeArray;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```

Reply:
oo good to know!